### ISSUE TITLE
Test set

### ISSUE THREAD

**Issue by zhangyahu1** [HuangZhiChao95/FewShotMotionTransfer#2]
Thanks for sharing your code!
It is possible to release part of test samples for one subject?

**Comment by HuangZhiChao95** [comment]
Sorry. I cannot release the dataset because of the copyright issue. The copyright of the youtube videos is held by their uploader and I cannot redistribute them. I give the script and list to process the test and training data under data_preprocess/ and video_list.txt.

**Comment by zhangyahu1** [comment]
Thanks for your response.
